AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -22,53 +25,76 @@ extern BOOL CGPointIsNull(CGPoint point)
   return isnan(point.x) && isnan(point.y);
 }
 
+@interface ASLayout ()
+
+/**
+ * A boolean describing if the current layout has been flattened.
+ */
+@property (nonatomic, getter=isFlattened) BOOL flattened;
```

Comment:
Flatten is now private, as it's only needed in the flattening traversal process.